packages/node/src/index.ts

```diff
@@ -381,7 +381,10 @@ export const build: BuildV3 = async ({
 };
 
 export const prepareCache: PrepareCache = async ({ workPath }) => {
-  const cache = await glob('node_modules/**', workPath);
+  const cache = {
```

Comment:
@TooTallNate @styfle - i didn't see a test for this block. Any ideas how I can test it? This package has an integration test folder so i was thinking like maybe setting up a fake monorepo (i.e. with a root `node_modules` and a sub-directory `node_modules`). WDYT?

Reply:
I think the same style of unit tests for static-build are best

Reply:
I suggest taking inspiration from [here](https://github.com/vercel/vercel/blob/main/packages/static-build/test/prepare-cache.test.ts) (that is, unit test the `prepareCache()` function).